src/legacy/core_plugins/embeddable_api/public/np_core.test.mocks.ts

```diff
@@ -17,33 +17,15 @@
  * under the License.
  */
 
-import { fatalErrorsServiceMock, notificationServiceMock } from '../../../../core/public/mocks';
+import { coreMock } from '../../../../core/public/mocks';
```

Comment:
What do you think about having this file also be in NP so we don't even have to duplicate this everywhere?  Then in your actual test files, you'd just import something like 

`import { coreMock } from '../../../../core/public/np_core.test.mocks';`?

... though I suppose in this case I wouldn't want to mock the overlay functions for everyone...  But it does seem like in most cases, just importing the single file is enough for folks who don't have to mock anything particular.  🤔  Just a thought!